src/map/script.c

```diff
@@ -12618,13 +12623,49 @@ BUILDIN(waitingroomkickall) {
 	return true;
 }
 
+/// Kicks target player from the waiting room of the current or target npc.
+///
+/// kickwaitingroom "<npc_name>"{,"<name>"|<account id>};
+/// kickwaitingroom;
+BUILDIN(waitingroomkick) {
```

Comment:
see [coding guidelines chapter 3](https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-3-placing-braces-and-spaces)

>However, there is one special case, namely functions: they have the opening brace at the beginning of the next line, thus:

	int function(int x)
	{
		body of function
	}

PS: yeah I didn't follow this guideline in my plugins :P